js/src/logger.ts

```diff
@@ -305,6 +306,15 @@ export class BraintrustState {
   private _apiConn: HTTPConnection | null = null;
   private _proxyConn: HTTPConnection | null = null;
 
+  public promptCache = new PromptCache({
+    cacheDir:
+      iso.getEnv("BRAINTRUST_PROMPT_CACHE_DIR") ??
+      "/tmp/braintrust/prompt_cache",
```

Comment:
better to use `$HOME/.braintrust/cache`, because the OS is allowed to wipe temporary directory files at any point. Also, it's not platform-safe to use `/tmp` (there's a portable tmpdir library in JS that you can use, but you should put that logic in the node implementation).

Reply:
Done